lib/sidekiq/group/collection.rb

```diff
@@ -58,13 +58,11 @@ def success(jid)
       def remove_processed(jid)
         Sidekiq::Logging.logger.info "Child job #{jid} completed" if Sidekiq::Group.debug
 
-        unless Sidekiq.redis { |r| r.srem("#{@cid}-jids", jid) }
-          Sidekiq::Logging.logger.info "Could not remove child job #{jid} from Redis" if Sidekiq::Group.debug
+        return if Sidekiq.redis { |r| r.srem("#{@cid}-jids", jid) }
```

Comment:
You mentioned it returns `0`. It is truthy value in ruby.

Reply:
right, Redis returns 1 or 0, but ruby Redis converts it to Boolean

https://www.rubydoc.info/github/redis/redis-rb/Redis:srem

Reply:
<img width="485" alt="image" src="https://user-images.githubusercontent.com/3591865/132315901-49a73b00-b397-4201-82ba-1b2de9903951.png">